code/modules/mob/emote.dm

```diff
@@ -1,32 +1,39 @@
 //The code execution of the emote datum is located at code/datums/emotes.dm
 /mob/proc/emote(act, m_type = null, message = null, intentional = FALSE)
+	if(emote_timer == world.time && intentional)
```

Comment:
This logic seems wrong. Shouldn't it be `>=` and `!intentional`?

Reply:
This logic feels ambiguous, but maybe the compiler is fine with it.

Ike's comment is at least 100% correct.
```suggestion
	if(emote_timer >= world.time && intentional)
```

Reply:
Uuuh... hold on let me grab a bottle of vodka and think about this for a moment.

Reply:
```if intentional
    return TRUE
if(emote_timer >= world.time)
    ...
    ```

Reply:
Actually... you _already_ check if the emote was intentional in the "check_cooldown" proc? Why check it again here?

Reply:
Hmmmmmmmm....